s_tui/s_tui.py

```diff
@@ -263,14 +283,16 @@ def update_displayed_information(self):
         self.clock_view.set_text(('bold text', seconds_to_text(
             int(self.controller.stress_time))))
 
+    # pycharm warning suppression for unused vars in function
```

Comment:
Is this a PyCharm byproduct?

Reply:
pycharm gives weak warnings for some stuff, i added some comments to suppress all of those.
this isn't the only instance of this, i can remove them if it bothers you